couchbase/src/api/results.rs

```diff
@@ -169,6 +170,49 @@ pub struct AnalyticsMetaData {
     client_context_id: String,
 }
 
+#[derive(Debug, Deserialize)]
+pub struct SearchMetaData {
+    errors: Option<HashMap<String, String>>,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct SearchRow {
```

Comment:
Is this used at all in your code, if you turn it right into a json value ?

Reply:
As discussed, used in the result.